make_test_refs.py

```diff
@@ -0,0 +1,34 @@
+from datasets import ClassLabel, load_dataset, Value 
+import torch
+
+
+if __name__=='__main__':
+    """
+    Creates references for evaluation and stores MRs into a file (input for a trained generative model)
+    """
+
+    data = load_dataset('e2e_nlg')['test']
+    dictic = {}
```

Comment:
what does `dictic` mean? WOuld be nice to add a type hint. Based on your code below, it should be:

```
dictic: Dict[str, List[str]] = {}
```